labelspark/__init__.py

```diff
@@ -95,27 +134,37 @@ def bronze_to_silver(bronze_table):
             print("No classifications found.")
 
         # object counting
-        try:  #this field won't work if the Label does not have objects in it
+        try:  # this field won't work if the Label does not have objects in it
             for object in row.get("Label.objects.title", []):
                 object_name = '{}.count'.format(object)
                 if object_name not in my_dictionary:
-                    my_dictionary[object_name] = 1  #initialize with 1
+                    my_dictionary[object_name] = 1  # initialize with 1
                 else:
-                    my_dictionary[object_name] += 1  #add 1 to counter
+                    my_dictionary[object_name] += 1  # add 1 to counter
         except Exception as e:
             print("No objects found.")
 
         my_dictionary["DataRowID"] = row.DataRowID  # close it out
+        if video:
+            my_dictionary[
+                "frameNumber"] = row.frameNumber  # need to store the unique framenumber identifier for video
         new_json.append(my_dictionary)
 
     parsed_classifications = pd.DataFrame(new_json).to_spark()
+    print(parsed_classifications)
 
     bronze_table = bronze_table.to_spark()
-    joined_df = parsed_classifications.join(bronze_table, ["DataRowID"],
-                                            "inner")
-    joined_df = joined_df.withColumnRenamed("DataRowID", "DataRow ID")
+    if video:
```

Comment:
I would prob recommend breaking out asset type related functionality into seperate functions, otherwise the code will get messy if you have to sprinkle asset specific code throughout the function

Reply:
We wanted to have as simple and few method calls as possible so the consumer doesn't have to worry about the asset type. But we can consider this for future too.